src/common/format.js

```diff
@@ -63,3 +63,12 @@ export function formatMilliseconds(duration) {
 
   return response;
 }
+
+/*
+ * Formats a floating number to be a String expressed as seconds in x (default 1 decimal) decimals.
+ * Ex: 15.6664 => 15.7
+ */
+export function formatSeconds(seconds, precision=1) {
```

Comment:
Can you explain how this helper is any different from just using `toFixed` ?

Reply:
Ah, didn't know about `toFixed`. Seems pretty similar, however this helper does not add additional nulls. 
0.toFixed(2) = 0.00
formatSeconds(0) = 0

Very minor thing though.

Reply:
Personally I think the fixed number of decimal places is the desired behavior, and that this formatting helper is superfluous.

If you really think the cut decimal place behavior is better and you want to keep the function, I ask that you rename it because currently it has nothing to do with 'seconds'.

Reply:
Alright, I will remove the function from this file and use `.toFixed` inside the style switch statement instead.